src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.macOS.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Uno.Extensions;
+using Uno.UI;
+
+using Foundation;
+using AppKit;
+using CoreGraphics;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class ItemsControl : Control
+	{
+		partial void InitializePartial()
+		{
+
+		}
+
+		partial void RequestLayoutPartial()
+		{
+			NeedsLayout = true;
```

Comment:
Use `InvalidateMeasure()` instead, `NeedsLayout` does not propagate the invalidation to the parents.

Reply:
Fixed